run.go

```diff
@@ -325,6 +325,15 @@ func (r *Run) request(ctx context.Context, payload any) (err error) {
 
 						done, _ = out["done"].(bool)
 						r.rawOutput = out
+					case []any:
```

Comment:
Is this checking if credential returns an array of objects? if so then it seems fine to me